### PR TITLE
Add dotnet hello world task and webapp

### PR DESCRIPTION
**Pull request by JanakaSandaruwan**
## Purpose
pram https://github.com/wso2-enterprise/choreo/issues/26117

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: hello-world-dotnet-task/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Hello World .NET
+
+Sample for Hello World task. Print "Hello World" when run the main.py
```
Comment (ThusharaSampath) [inline hello-world-dotnet-task/readme.md:3]: lets update this

File: hello-world-dotnet-task/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    | **Field**             | **Description**                               |
+    |-----------------------|-----------------------------------------------|
+    |Name           | Hello World Dotnet Task              |
+    |Description    | Hello World Dotnet Task       |
+    | **GitHub Account**    | Your account                                  |
+    | **GitHub Repository** | choreo-samples |
+    | **Branch**            | **`main`**                               |
+    | **Buildpack**      | .NET |
+    | **Select Go Project Directory**       | hello-world-python-task |
```
Comment (ThusharaSampath) [inline hello-world-dotnet-task/readme.md:23]: lets update this as well

File: .samples/hello-world-dotnet-task.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+displayName: Hello World in .NET
+description: Print Hello World message
+componentType: manual-task
+buildPack: dotnet
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /hello-world-dotnet-task
+thumbnailPath: /icons/hello-world-dotnet-task.png
+documentationPath: /hello-world-dotnet-task/readme.md
+tags: null
```
Comment (chameerar) [inline .samples/hello-world-dotnet-task.yaml:9]: Please add some tags

File: hello-world-dotnet-task/readme.md (hunk 1 above)
Comment (JanakaSandaruwan) [reply hello-world-dotnet-task/readme.md]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/8c0bf8690a2860b6f806d4bcad197fe86f1792c7

File: hello-world-dotnet-task/readme.md (hunk 2 above)
Comment (JanakaSandaruwan) [reply hello-world-dotnet-task/readme.md]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/8c0bf8690a2860b6f806d4bcad197fe86f1792c7

File: .samples/hello-world-dotnet-task.yaml (hunk 3 above)
Comment (JanakaSandaruwan) [reply .samples/hello-world-dotnet-task.yaml]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/8c0bf8690a2860b6f806d4bcad197fe86f1792c7

File: .samples/hello-world-dotnet-task.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+displayName: Hello World in .NET
+description: Print Hello World message
```
Comment (chameerar) [inline .samples/hello-world-dotnet-task.yaml:2]: Let's make the description more meaningful. 
eg - Task to print "Hello World" in .NET

File: .samples/hello-world-dotnet-webapp.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+displayName: Hello World Web App in .NET
+description: Print Hello World message
```
Comment (chameerar) [inline .samples/hello-world-dotnet-webapp.yaml:2]: Let's make the description more meaningful.

File: hello-world-dotnet-task/readme.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Hello World .NET
```
Comment (chameerar) [inline hello-world-dotnet-task/readme.md:1]: Let's change the title to be more descriptive.
eg - Hello World task .NET

File: hello-world-dotnet-webapp/readme.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+# Hello World .NET
```
Comment (chameerar) [inline hello-world-dotnet-webapp/readme.md:1]: Let's make the Readme title more descriptive

File: .samples/hello-world-dotnet-task.yaml (hunk 4 above)
Comment (JanakaSandaruwan) [reply .samples/hello-world-dotnet-task.yaml]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/38110a68153068ccabca2a2bdf30142179c2215f

File: .samples/hello-world-dotnet-webapp.yaml (hunk 5 above)
Comment (JanakaSandaruwan) [reply .samples/hello-world-dotnet-webapp.yaml]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/38110a68153068ccabca2a2bdf30142179c2215f

File: hello-world-dotnet-webapp/readme.md (hunk 7 above)
Comment (JanakaSandaruwan) [reply hello-world-dotnet-webapp/readme.md]: Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/38110a68153068ccabca2a2bdf30142179c2215f

File: .samples/icons/hello-world-dotnet-webapp.png
Comment (chameerar) [inline .samples/icons/hello-world-dotnet-webapp.png:1]: Better to add a proper icon. You can request a one from the UX team.

Comment (JanakaSandaruwan) [reply .samples/icons/hello-world-dotnet-webapp.png:1]: Yes. For other buildpack samples too uses the same icon. I will request and later update the icon

**Review by chameerar — APPROVED** [review]

File: .samples/icons/hello-world-dotnet-webapp.png
Comment (chameerar) [reply .samples/icons/hello-world-dotnet-webapp.png:1]: Ack

**Pull request merged**